onnxruntime/core/providers/cpu/nn/tfidfvectorizer.cc

```diff
@@ -412,31 +390,45 @@ Status TfIdfVectorizer::Compute(OpKernelContext* ctx) const {
   }
 
   assert((num_rows * C) == total_items);
-  // Frequency holder allocate [B..output_size_]
-  // and init all to zero
-  std::vector<uint32_t> frequencies;
-  frequencies.resize(num_rows * impl_->output_size_, 0);
+  const Impl& impl = *impl_;
+  std::vector<int64_t> output_dims;
```

Comment:
We need to change it to TensorShapeVector

Reply:
done

Reply:
Not seeing the change.